src/cplus_plugin/gui/constant_rasters/constant_raster_manager_dialog.py

```diff
@@ -810,7 +894,11 @@ def _ensure_checked_components_in_collection(self):
         # Get the current value from the widget to use for ALL checked items
         current_value = None
         if hasattr(current_config_widget, "sb_experience"):
```

Comment:
@osundwajeff looks can we remove the years of experience here as the dialog should be generic to handle all use cases of constant raster types. Any constant raster type UI details should be handled inside the configuration widget in order to separate concerns.